jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/cmds/FusekiMain.java

```diff
@@ -442,6 +448,15 @@ protected void processModulesAndArgs() {
 
         // -- Server setup.
 
+        if ( contains(argPathBase) ) {
+            // Static files.
+            String servletContextPath = getValue(argPathBase);
+
+            if ( servletContextPath.endsWith("/") )
```

Comment:
This check shouldn't trigger on a plain `/` as that's a valid value for the root context

Reply:
Fixed